internal/shim-sgx/Cargo.toml

```diff
@@ -28,6 +28,7 @@ xsave = { version = "2.0.2" }
 rcrt1 = "1.0.0"
 lset = "0.2"
 sgx = "0.3"
+mmledger = { git = "https://github.com/enarx/mmledger" }
```

Comment:
Please use git hash references, so, that if the remote branch gets updated, the enarx builds don't break.